master/pkg/model/experiment.go

```diff
@@ -857,15 +857,17 @@ func ExitedReasonFromProto(r trialv1.TrialEarlyExit_ExitedReason) ExitedReason {
 	}
 }
 
-// ToProto converts an ExitedReason to its protobuf representation.
-func (r *ExitedReason) ToProto() experimentv1.TrialExitedEarly_ExitedReason {
+// ToSearcherProto converts an ExitedReason to its protobuf representation for searcher purposes.
+func (r *ExitedReason) ToSearcherProto() experimentv1.TrialExitedEarly_ExitedReason {
 	switch *r {
```

Comment:
not sure why this has a pointer receive, just to deref it

Reply:
A case of cargo-culting by a golang amateur :)